tests/FSharp.Compiler.ComponentTests/EmittedIL/CompiledNameAttribute/CompiledNameAttribute04.fs.il.net472.debug.bsl

```diff
@@ -142,8 +142,7 @@
     .method public hidebysig virtual final 
             instance int32  CompareTo(valuetype Program/S obj) cil managed
     {
```

Comment:
What happened to: 
DebuggerNonUserCodeAttribute()

Although:
CompilerGeneratedAttribute + DebuggerNonUserCodeAttribute does seem slightly redundent